File: Exercico2/ControleDeAudio.py
class ControleDeAudio:
    def __init__(self):
        self.volume = 2

    def aumentar_volume(self):
        if self.volume < 10:
            self.volume += 1

    def diminuir_volume(self):
        if self.volume > 0:
            self.volume -= 1

    def ler_volume(self):
        return self.volume

File: Exercico2/ControleDeAudio.ts
class ControleDeAudio {
    private volume: number = 2;

    aumentarVolume() {
        if (this.volume < 10) {
            this.volume++;
        }
    }

    diminuirVolume() {
        if (this.volume > 0) {
            this.volume--;
        }
    }

    lerVolume(): number {
        return this.volume;
    }
}